manifests/state-spectrum-x-operator/0050-flowcontroller-ds.yaml

```
@@ -38,9 +38,11 @@ spec:
             - /flowcontroller
           args:
             - --cm-namespace={{ .RuntimeSpec.Namespace }}
+            {{- if .CrSpec.SpectrumXConfig }}
```

nit. Combine conditions
```suggestion
     {{- if and .CrSpec.SpectrumXConfig .CrSpec.SpectrumXConfig.Name }}
- --cm-name={{ .CrSpec.SpectrumXConfig.Name }}
{{- end }}
```

---

Done